aries-site/src/pages/components/namevaluelist.mdx

```diff
@@ -202,6 +207,92 @@ For vertical lists:
   <NameValueListReadMoreExample />
 </Example>
 
+## Name-Value Pair with Empty Value
+
+If the value of a name-value pair is empty, use `--` in place of where a value would appear. This visual indicator aids the user in identifying fields that do not have values without increasing cognitive load.
```

Comment:
```suggestion
If the value of a name-value pair is empty, use `--` in place of where a value would appear. This visual indicator aids the user in identifying fields that do not have values, without increasing cognitive load.
```

Reply:
I'd maybe add a comma here to clarify the separation between 'values' and 'without'.